airbyte-webapp/build.gradle

```diff
@@ -46,8 +52,15 @@ task pnpmBuild(type: PnpmTask) {
 
     inputs.files commonConfigs
     inputs.file '.eslintrc.js'
+    inputs.file '.stylelintrc'
+    inputs.file 'scripts/calculate-source-hash.sh'
+    inputs.file 'scripts/load-declarative-schema.sh'
```

Comment:
Maybe worth adding the whole `scripts` folder? There are some dev scripts in there that don't matter, but I can imagine we will add some other build-relevant scripts there later and forget to add them here.